cli/admin/call.go

```diff
@@ -173,12 +173,12 @@ func printCallRes(callResRaw interface{}) {
 
 	switch callRes := callResRaw.(type) {
 	case string:
-		log.Info(callRes)
+		fmt.Println(callRes)
 	case []interface{}:
 		for _, callResLineRaw := range callRes {
 			switch callResLine := callResLineRaw.(type) {
 			case string:
-				log.Info(callResLine)
+				fmt.Println(callResLine)
 			default:
 				needReturnValueWarn = true
 				log.Infof("%v\n", callResLine)
```

Comment:
I thought that the tool outputs a dot in case of errors, and then it turns out a kind of error. Well, if a person does not return a string, he clearly does not care about formatting the output.

Reply:
I disagree. i think we should output what is transferred from the tarantool without additional logger wrappers.

Reply:
Now the message is:
```
   • [not string 666]
        
   • Admin function should return string or string array value
```

Need to do this?
```
[not string 666]
        
   • Admin function should return string or string array value
```

Reply:
It doesn't look very consistent

Reply:
Code:
```
    local test = {
        usage = 'test',
        args = {
        },
        call = function()
            return {
                "Hello bro", 2
            }
        end,
    }
```
Consistent (IMHO):
```
└──╼ cartridge admin test --name myapp --run-dir tmp/run/
Hello bro
2
   • Admin function should return string or string array value
```
Not consistent:
```
└──╼ cartridge admin test --name myapp --run-dir tmp/run/
Hello bro
   • 2
                       
   • Admin function should return string or string array value

```

Reply:
OK) I don't mind, I'll fix it now

Reply:
done